app/operations/post_test.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+
+from aidbox_python_sdk.types import SDKOperation, SDKOperationRequest
+from aiobotocore.session import get_session
+from aiohttp import web
+
+from app import config
+from app.sdk import sdk
+
+
+@sdk.operation(["POST"], ["$get-presigned-urls"])
+async def generate_presigned_urls_op(
+    _operation: SDKOperation, request: SDKOperationRequest
+) -> web.Response:
+    bucket = "dataintake"
+    filename = "file"
```

Comment:
accept filename from the resource, it will make the file URL is more human readable.
e.g.
```
POST /$get-presigned-url

filename: myfile.jpg
```

also it might be optional. Read about https://github.com/Aidbox/aidbox-python-sdk?tab=readme-ov-file#usage-of-request_schema